src/components/markdown/CodeBlock/CodeBlock.tsx

```diff
@@ -102,6 +102,8 @@ class CodeBlock extends React.Component<
 
   constructor(props) {
     super(props);
+    const isCodeBlockExpandable = props.isCodeBlockExpandable ?? true;
```

Comment:
oh whoops I didn't realize that this was a class component because github collapsed the lines. This line should be moved down into the render function.

Reply:
Fixed.